### DISCUSSION TITLE
正版登錄多人模式下無法看到自己跟别人的皮肤

### DISCUSSION THREAD

**Discussion by xpr224** [Meloong-Git/PCL#2255] [提问] [answered]
在伺服器里看所有人都是史蒂夫，包括自己
但是单人模式看自己则有皮肤 这个有什么解决方案吗

**Comment by Pigeon0v0** [comment, marked as answer]
开启服务器正版验证

**Comment by PlayMcBKuwu** [comment]
> 在伺服器里看所有人都是史蒂夫，包括自己 但是单人模式看自己则有皮肤 这个有什么解决方案吗

你玩 / 開的伺務器中 **是離綫驗證** 或者 **沒有 SkinsRestorer** 插件。
解決方案：
- 開啓正版驗證。
- 使用 **Spigot / Paper** 等插件端安裝 **SkinsRestorer** 插件。
